scripts/cardano-node.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Hmm, I wonder if there's a nix-based version of this we can use? I'm surprised if there isn't something like this somewhere else already, and that might require more straightforward maintenance than this one. Then again, I don't *actually* want to have a "real" dependency on the node, so I'm not sure what the best thing to do is. Maybe this is fine for now and we can worry about it if it becomes hard to maintain.

Reply:
Yeah, you can run the node by checking out cardano-node, but I agree having to depend on the cardano-node repo isn't ideal.